tests/missing/test_imputation.py

```diff
@@ -0,0 +1,70 @@
+#Code to test the imputation strategies.
+from probatus.missing.imputation import CompareImputationStrategies
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.linear_model import LogisticRegression
+from sklearn.experimental import enable_iterative_imputer  
+from sklearn.impute import KNNImputer,SimpleImputer,IterativeImputer
+import pandas as pd
+import numpy as np
+import pytest
+import os
+
+@pytest.fixture(scope='function')
+def X():
+    return pd.DataFrame({'col_1': [1, np.nan, 1, 1, np.nan, 1, 1, 0],
+                         'col_2': [0, 0, 0, np.nan, 0, 0, 0, 1],
+                         'col_3': [1, 0, np.nan, 0, 1, np.nan, 1, 0], 
+                         'col_4': ['A', 'B', 'A', np.nan, 'B', np.nan, 'A', 'A']}, index=[1, 2, 3, 4, 5, 6, 7, 8])
+
+@pytest.fixture(scope='function')
+def y():
+    return pd.Series([1, 0, 1, 0, 1, 0, 1, 0], index=[1, 2, 3, 4, 5, 6, 7, 8])
+
+def test_imputation_linear(X,y,capsys):
+    
+   #Create strategies for imputation.
+   strategies = {
```

Comment:
You can make this a fixture as well

Reply:
Done.

Reply:
Here i meant the strategies variable only.

Reply:
Got it, done.